README.md

```diff
@@ -278,7 +290,66 @@ Installation Instructions
        callhome_group1: [host-vm1,host-vm2,host-vm3,host-vm4]
        callhome_schedule: [daily,weekly]
      ```
-
+     
+  4. To Install and configure Protocol Service (SMB and NFS) in the cluster you'll need to provide additional information. It is recommended to use [group variables](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#assigning-a-variable-to-many-machines-group-variables) as follows:
+       
+       **IPv4 CES inventory:**
+       ```yaml
+       # group_vars/all.yml:
+       ---
+       scale_protocols:
+           smb: false
+           nfs: false
+           export_ip_pool: [192.168.100.100,192.168.100.101]
+           filesystem: cesSharedRoot
+           mountpoint: /gpfs/cesSharedRoot
+       ```
+     
+       ```yaml
+       smb: set True for required protocol
+       nfs: set True for required protocol
+       export_ip_pool: Comma separated list of ipv6 CES IP
+       filesystem: Any file system that is going to act as cesSharedRoot filesystem
+       mountpoint: CES shared root file system mount point.
+       ```
+     
+       **IPv6 CES inventory:**
+       ```yaml
+       # group_vars/all.yml:
+       ---
+       scale_protocols:
+            smb: false
+            nfs: false,
+            interface: [eth0]
+            export_ip_pool: [2002:90b:e006:84:250:56ff:feb9:7787]
+            filesystem: cesSharedRoot
+            mountpoint: /gpfs/cesSharedRoot
+       ```
+     
+      User have to set True` for required protocol **smb** and or **nfs**
```

Comment:
I don't understand this statement... isn't it the same as with IPv4 (ln. 309 - 310)?